component/custom/custom_loader.go

```diff
@@ -0,0 +1,40 @@
+package custom
+
+import (
+	"os"
+	"plugin"
+	"strings"
+)
+
+func init() {
+	pluginPath := os.Getenv("PLUGIN_PATH")
+	if pluginPath == "" {
+		return
+	}
+
+	// environment variable can contain multiple paths separated by colon.
+	paths := strings.Split(pluginPath, ":")
```

Comment:
Rather than this, we can mandate a directory where all plugins should be placed and indicate it as BLOBFUSE_PLUGIN_DIRECTORY

Reply:
The custom component can just list the *.so files in that directory